src/config/example.toml

```diff
@@ -169,6 +169,7 @@ stream_enable_shared_source = true
 stream_switch_jdbc_pg_to_native = false
 stream_max_barrier_batch_size = 1024
 stream_hash_join_entry_state_max_rows = 30000
+stream_enable_explain_analyze_stats = true
```

Comment:
@kwannoel If we intend this feature to be used by users, shall we not include the settings in the `developer` section? Sorry for the belated comment. I was looking at the blog post about `EXPLAIN ANALYZE`. I feel this is an awesome feature and experienced users will find it super useful. 
cc: @hzxa21